modules/rosetta/src/main/scala/org/tessellation/rosetta/server/model/search/SearchTransactionRequest.scala

```diff
@@ -0,0 +1,14 @@
+package org.tessellation.rosetta.server.model.search
+
+/** This is an intermediary object used to hold the values from the Rosetta request that is relevant to our system.
+  */
```

Comment:
if this is an intermediary object, shouldn't it hold our codebase types? what is the purpose of this case class and how it will be used?

Reply:
The purpose of this case class is to hold data extracted from the request and to be passed around to different methods that need that data. Basically, it holds the parameters for various methods. Where is the code base types?

Reply:
ok, sure. Is there no specification in the openapi so that the class can be auto-generated?
By codebase types I mean for example `Address` or `DAGAddress` instead of `String`, or `Hash` instead of `String`.

Reply:
I'll confer with @tbekas and @ryle-goehausen on this.

Reply:
After discussing with @ryle-goehausen , I'll go ahead and add codebase types to this class.

Reply:
Refined types and code base types have been added to the class as appropriate.